src/store/ChartStore.js

```diff
@@ -210,10 +210,8 @@ class ChartStore {
 
         const deleteElement = stxx.chart.panel.holder.parentElement.querySelector('#mouseDeleteText');
         const manageElement = stxx.chart.panel.holder.parentElement.querySelector('#mouseManageText');
-        const manageTouchElement = stxx.chart.panel.holder.parentElement.querySelector('#overlayTrashCan');
         deleteElement.textConent = t.translate('right-click to delete');
         manageElement.textConent = t.translate('right-click to manage');
-        manageTouchElement.textContent = t.translate('tap to manage');
```

Comment:
@brucebinary cause we show the delete icon in mobile instead of `tap to manage`  as per new @afiq design.
![image](https://user-images.githubusercontent.com/3080372/43830866-90e069ba-9b17-11e8-883c-c87fcf099955.png)

Reply:
@MahboobehMohammadi ah ok. Btw there are some console errors when using the draw tools. If it's nothing too hard do try to fix it in this PR. I suspect it's cause by some refactoring I did to please eslint. Feel free to just use

`// eslint-disable-line react/no-array-index-key`